src/altinn-app-frontend/src/features/form/containers/RepeatingGroupTable.tsx

```diff
@@ -451,22 +452,24 @@ export function RepeatingGroupTable({
                           return (
                             <TableCell key={`${component.id}-${index}`}>
                               <span>
-                                {getFormDataForComponent(component, index)}
+                                {index !== editIndex
+                                  ? getFormDataForComponent(component, index)
+                                  : null}
                               </span>
                             </TableCell>
                           );
                         })}
                         <TableCell
                           align='left'
-                          style={{ padding: 0 }}
+                          style={{ width: '110px', padding: 0 }}
```

Comment:
I haven't tested, but how does this affect different labels/languages for the button text? See #469 for an example. Not sure if a fixed width improves anything here.

Reply:
The reason i added the hardcoded width in this instance was to make the cells for the buttons behave the same way when the table-header is not displayed. The width is inherited from the column-header and needed to be set on the cells in order to retain the width when the header is not displayed. This will unfortunately not improve the issue in #469.

Reply:
Ah - I get it. The way I see it, this problem has been documented well enough in #469 then.

Just a quick comment, though - could `min-width` (or `minWidth`) work better here instead of a fixed `width`? That way it can theoretically scale to become wider, at least.